src/components/Board.vue

```diff
@@ -9,6 +9,9 @@
         <p class="name">{{ individual.username }}</p>
         <p class="points">{{ individual.blocksCompleted }}</p>
       </div>
+      <h3 v-if="individualsWithRank.length < 1">
+        There's no leaderboard quite yet, check back soon!
```

Comment:
This is if there's not enough people on a team for there to be a leaderboard right?

Reply:
Yea, the endpoint will only start returning data when there are teams and probably block records